ipn/ipnlocal/network-lock.go

```diff
@@ -280,7 +281,10 @@ func (b *LocalBackend) tkaBootstrapFromGenesisLocked(g tkatype.MarshaledAUM) err
 		return fmt.Errorf("reading genesis: %v", err)
 	}
 
-	chonkDir := b.chonkPath()
+	chonkDir := b.chonkPathLocked()
```

Comment:
Nit: is there are reason we aren't doing `MkDirAll` ?

Reply:
No real reason, I was just unsure if I wanted the `TailscaleVarRoot` be created with `0755`. Elsewhere it is created with 0700.